app/api-v1/routes/recipe.js

```diff
@@ -5,7 +5,27 @@ module.exports = function (recipeService) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      if (!req.body) {
+        return res.status(400).json({ message: 'Invalid request' })
+      }
+
+      const { id, externalId, name, imageAttachmentId, material, alloy, price, requiredCerts, supplier } = req.body
+
+      const recipe = await recipeService.createRecipe({
+        id,
+        externalId,
+        name,
+        imageAttachmentId,
+        material,
+        alloy,
+        price,
+        requiredCerts,
+        supplier,
+      })
+      if (recipe == null) {
+        return res.status(200).json({ message: 'An error occurred' })
+      }
+      res.status(201).json(recipe)
     },
```

Comment:
At some point it'll need a swagger schema like this one:
https://github.com/digicatapult/wasp-thing-service/blob/main/app/api-v1/routes/thing.js#L68

Reply:
um it has one? See ln 49

Reply:
Yep ignore this

Reply:
thanks guys, wasn't sure if this was generated or not? I'm presuming it has come from somewhere